spec/models/review_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+require_relative "../../app/models/review.rb"
+
+RSpec.describe Review, :type => :model do
+  it "is not valid without a rating" do
+    review = Review.new(rating: nil, body: "It's a party for the palette", price_range: "$$", photo_url: "pic.u")
+    expect(review).to_not be_valid
+    expect(review.errors.messages).to eq(:rating=>["can't be blank"])
+  end
```

Comment:
This testing is good and sufficient! However, it could be made more readable using a slightly different syntax via "shoulda_matchers":
```
it { should have_valid(:rating).when(5) }
it { should_not have_valid(:rating).when(nil) }
```

Reply:
noted! we will circle back to this later and use for future reference. thank you!

Reply:
Reference is here :) https://learn.launchacademy.com/lessons/sinatra-testing-activerecord-validations-associations